frontend/www/js/omegaup/components/login/Signup.vue

```diff
@@ -106,31 +116,75 @@
 </template>
 
 <script lang="ts">
-import { Vue, Component, Prop } from 'vue-property-decorator';
+import { Vue, Component, Prop, Watch } from 'vue-property-decorator';
 import omegaup_Markdown from '../Markdown.vue';
 import T from '../../lang';
-
+import * as time from '../../time';
+import DatePicker from '../DatePicker.vue';
+import * as ui from '../../ui';
 @Component({
   components: {
     'omegaup-markdown': omegaup_Markdown,
+    'omegaup-datepicker': DatePicker,
   },
 })
 export default class Signup extends Vue {
   @Prop() validateRecaptcha!: boolean;
-
   T = T;
   username: string = '';
-  email: string = '';
+  email: null | string = null;
+  parentEmail: null | string = null;
   password: string = '';
   passwordConfirmation: string = '';
   recaptchaResponse: string = '';
-
+  birthDate: null | Date = new Date(0);
+  checked = false;
```

Comment:
```suggestion
  privacyPolicyAccepted = false;
```

otherwise `checked` doesn't really tell us what's going on. this also needs to gate being able to click the button: a `get disabled(): bool { ... }` should check that the birth date is set, this is checked, and the other two fields have values.